recipe/build.sh

```diff
@@ -1,29 +1,72 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Is this a conda-forge style recommendation?

Reply:
I think it is, at least that's why i started using it in my recipes